app/Exceptions/Handler.php

```diff
@@ -34,7 +34,7 @@ public function render($request, Exception $exception)
                 return new JsonResponse([
                     $prev->getErrorCode(),
                     $prev->getMessage(),
-                    $prev->getTraceAsString()
+                    $prev->getTrace()
```

Comment:
For better debugging

Reply:
If we'are doing anything in this place we should check whether an env is PROD or DEV at least ... Could you add this checking point to disable displaying all debug information in prod?

Reply:
The `APP_DEBUG=false` in `.env` protects against that.
When set to `false`, api returns:
```json
{
    "message": "Server Error"
}
```